__tests__/app.test.js

```diff
@@ -123,3 +123,41 @@ describe('GET /api/articles', () => {
             });
     });
 });
+describe('GET /api/articles/:article_id/comments', () => {
+    test('should return a 200 status code', () => {
```

Comment:
Could you also add a test for articles that have no comments? This should also return a 200 status code